airflow/cli/commands/connection_command.py

```diff
@@ -162,7 +162,9 @@ def connections_export(args):
     """Export all connections to a file."""
     file_formats = [".yaml", ".json", ".env"]
     if args.format:
-        warnings.warn("Option `--format` is deprecated.  Use `--file-format` instead.", DeprecationWarning)
+        warnings.warn(
+            "Option `--format` is deprecated.  Use `--file-format` instead.", DeprecationWarning, stacklevel=3
```

Comment:
```suggestion
            "Option `--format` is deprecated. Use `--file-format` instead.", DeprecationWarning, stacklevel=3
```

I don’t think we use two spaces after the period anywhere else. This is likely a typo.